WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsListFragment.kt

```diff
@@ -76,7 +77,7 @@ class StatsListFragment : Fragment() {
         viewModel = ViewModelProviders.of(this, viewModelFactory)
                 .get(statsType.name, StatsListViewModel::class.java)
 
-        viewModel.start(statsType)
+        viewModel.start(activity.intent.getSerializableExtra(WordPress.SITE) as SiteModel, statsType)
```

Comment:
To avoid a crash when the activity is restored after being killed (and intent extras are empty), we should save the site in `onSaveInstanceState()` and restore them here.

```suggestion
        val site = if (savedInstanceState == null) {
            val nonNullIntent = checkNotNull(nonNullActivity.intent)
            nonNullIntent.getSerializableExtra(WordPress.SITE) as SiteModel
        } else {
            savedInstanceState.getSerializable(WordPress.SITE) as SiteModel
        }
        viewModel.start(site, statsType)
```